packages/modal/src/resources/attach-focus-attribute.ts

```diff
@@ -0,0 +1,16 @@
+import { customAttribute, inject } from "aurelia-framework";
+
+@inject(Element)
+@customAttribute('ux-attach-focus')
+export class AttachFocusAttribute {
+
+  public value: boolean | string;
+
+  constructor(private element: HTMLElement) {}
+
+  public attached() {
+    if (this.value === '' || this.value === 'attach-focus' || (this.value && this.value !== 'false')) {
```

Comment:
This value `attach-focus` here is not needed. 
```suggestion
    if (this.value === '' || (this.value && this.value !== 'false')) {
```